blog/secadvisor.md

```diff
@@ -0,0 +1,290 @@
+---
+title: How to use Security Advisor with Skydive
+layout: blog-post
+author: Kalman Meth
+date: 31/07/2019
+---
+
+
+The Security Advisor filters the flow data obtained from Skydive, performs a data transformation, and saves the information to an object store in compressed JSON format.
+This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
+
+Instructions to developers to deploy and run the Secruity Advisor can be found at https://github.com/skydive-project/skydive/tree/master/contrib/pipelines/secadvisor.
```

Comment:
Use markdown link:
```
[label](url)
```

Reply:
OK

Reply:
I tried to do this but I get an error since it crosses to another repository. So I restored the original path.

Reply:
did you try 

[readme](https://github.com/skydive-project/skydive/tree/master/contrib/pipelines/secadvisor/README.md)

Reply:
I added in the label. But the url refers to a file outside the current repository, so the relative path doesn't work, and we have to use the full path.

Reply:
ok